deeprank/features/Edesolv.py

```diff
@@ -0,0 +1,411 @@
+import warnings
+
+from pdb2sql import pdb2sql, interface
+from deeprank.features import FeatureClass
+from copy import deepcopy
+from io import StringIO
+import numpy as np
+
+try:
+    from Bio.PDB import PDBParser, SASA
```

Comment:
It seems PDBParser and SASA could be replaced by pdb2sql and freesasa (already used in this package), repectively.

Reply:
freeSASA was constitutively giving problems when used at atom level: differently from residue level, the atom IDs are not always the same between freesasa and pdb2sql. That is why I decided to go for one single package to do the parsing and the SASA calculation.